documentation/modules/configuring/con-config-mirrormaker2.adoc

```diff
@@ -76,97 +76,121 @@ For more information, see xref:con-high-volume-config-properties-{context}[Handl
 .Example `KafkaMirrorMaker2` custom resource configuration
 [source,yaml,subs="+attributes"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaMirrorMaker2ApiVersion}
-kind: KafkaMirrorMaker2
+kind: KafkaMirrorMaker2 # <1>
 metadata:
   name: my-mirror-maker2
+# Deployment specifications
 spec:
+  # Kafka version (essential)
```

Comment:
Should be recommended or optional?

Reply:
We can go recommended, as with the Kafka config.